src/Akka.Persistence.Azure/Journal/HighestSequenceNrEntry.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Akka.Persistence.Azure.Journal
+{
+    internal sealed class HighestSequenceNrEntry
```

Comment:
THANK YOU for doing this - we needed a metadata table in order to get this functionality to work correctly. I appreciate you taking the time and effort to do this.